src/api/app/assets/javascripts/webui2/datatables.js

```diff
@@ -19,7 +19,9 @@ function initializeRemoteDatatable(cssSelector, params) { // jshint ignore:line
     processing: true,
     serverSide: true,
     ajax: $(cssSelector).data('source'),
-    pagingType: 'full_numbers'
+    pagingType: 'full_numbers',
+    stateSave: true,
```

Comment:
Should we also do this in the function `initializeDataTable`, so just above this one?

Reply:
True, if we use this we should do it on every DataTable